tasks/Containerfile

```diff
@@ -78,6 +78,7 @@ RUN groupadd -g 1111 -r user && useradd -r -g user -u 1111 user --home-dir /work
     echo 'user ALL=NOPASSWD: /usr/bin/chmod 666 /dev/kvm' > /etc/sudoers.d/user-fix-kvm
 
 ENV LANG=C.UTF-8 \
+    HOME=/work \
```

Comment:
That feels wrong.  This is going to have weird effects when we try to enter the container as `-u root`, for example.  Why doesn't this get set normally?

Checking this locally (against the current image, without this change):

```
$ podman run --rm quay.io/cockpit/tasks:latest sh -c 'echo $HOME'
/work
```

Reply:
Also: it occurs to me that now that we no longer have any "homedir path API" bindmounts, we could probably get rid of a great deal of the above special handling of creating `/work` and everything inside of it and let the homedir land in the default location.

Just about the only thing that looks actually useful is the gitconfig, and I have to think we could do something about that by setting some environment variables when we're running the bots... (or just set the environment variables on the container in the usual way when creating it)...